py-utils/src/utils/third_party/openldap/base_configure_ldap.py

```
@@ -0,0 +1,246 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+import os
+import ldap
+import glob
+from shutil import copyfile
+
+class BaseConfig:
+
+    def cleanUp():
+	    # Removing schemas
+        userSchemaFile = '/etc/openldap/slapd.d/cn=config/cn=schema/cn={1}s3user.ldif'
+        try:
+            os.remove(userSchemaFile)
+        except:
+            print('Error while deleting '+ userSchemaFile)
```

Raise exception instead of printing

---

done